Proiect in lucru

k4pn0 opened the pull request:
![432208300_409949121713752_6483801680432731799_n](https://github.com/ITSchool-Web-Wizards/Proiect-HTML-CSS/assets/155654169/9a142c13-1c77-4d89-9075-39d58bb7bf97)
![431741247_2451384051728778_1637173491791003538_n](https://github.com/ITSchool-Web-Wizards/Proiect-HTML-CSS/assets/155654169/6629723b-dcd6-440b-bc62-fc67a924fb90)
Am intampinat mai multe probleme
      - Nu am reusit sa fac meniul hamburger pana la capat; am urmat un video pe youtube si la un moment dat ceva nu a mai mers
      - Am incercat sa fac latura de sus a div-ului cu silueta in main si footer. In main am incercat cu un png si acolo a murit iar in footer am folosit un svg, dar alinierea lui nu este corecta
      -pattern-ul din section (pe elementul asta voiam sa fac parallax si nu mi-a iesit inca) si imaginea care este in comment (m-am tot jucat cu position si overflow sa arate ca in design dar nu am reusit)
Nici celelalte doua pagini nu sunt intr-o stare prea buna.  Recunosc ca nu am ajuns in stadiul asta in incercarile mele de a invata front end si m-a coplesit un pic sau poate mi-am ales o tema complicata. Nu am stiut daca trebuie sa am mai multe fisiere css sau trebuia sa aranjez codul in unul singur.  Am subestimat timpul necesar temei alese si acum cred ca am lucrat haotic. Imi cer scuze. Daca ma poti ajuta cu feedback, trucuri, sfaturi etc iti multumesc anticipat. 
Atasat si designul pentru homepage

k4pn0 committed:
Proiect in lucru

Am intampinat mai multe probleme
      - Nu am reusit sa fac meniul hamburger pana la capat; am urmat un video pe youtube si la un moment dat ceva nu a mai mers
      - Am incercat sa fac latura de sus a div-ului cu silueta in main si footer. In main am incercat cu un png si acolo a murit iar in footer am folosit un svg, dar alinierea lui nu este corecta
      -pattern-ul din section (pe elementul asta voiam sa fac parallax si nu mi-a iesit inca) si imaginea care este in comment (m-am tot jucat cu position si overflow sa arate ca in design dar nu am reusit)
Nici celelalte doua pagini nu sunt intr-o stare prea buna. 
Recunosc ca nu am ajuns in stadiul asta in incercarile mele de a invata front end si m-a coplesit un pic sau poate mi-am ales o tema complicata. Nu am stiut daca trebuie sa am mai multe fisiere css sau trebuia sa aranjez codul in unul singur.  Am subestimat timpul necesar temei alese si acum cred ca am lucrat haotic. Imi cer scuze. Daca ma poti ajuta cu feedback, trucuri, sfaturi etc iti multumesc anticipat

vladutcornel commented on index.html:
```diff
@@ -0,0 +1,96 @@
+                <a href="#">Povestea Milka</a>
+            </nav>
+        </aside>
+        <a href="index.html"> <img src="/Proiect-HTML-CSS/images/milka-logo-purple.webp" alt="logo Milka" style="max-width: 30vw;"></a>
```
Mai bine pui adrese relative, care să nu conțină path-ul care apare în.

```suggestion
        <a href="index.html"> <img src="images/milka-logo-purple.webp" alt="logo Milka" style="max-width: 30vw;"></a>
```

vladutcornel commented on index.html:
```diff
@@ -0,0 +1,96 @@
+<body>
+    <div class="nav-wrap">
+        <label id="hamburger-menu">
+            <input type="checkbox"/>
```
S-ar putea să te ajute dacă pui checkbox-ul înaintea meniului

```html
<input type="checkbox" class="menu-trigger"/>
<aside class="sliding-menu">
```

Asta te poate ajuta să faci un selector pentru atunci când checkbox-ul este bifat.

```css
input.menu-trigger:checked + .sliding-menu {
  /* Meniul este deschis */
}
```

vladutcornel commented:
> am urmat un video pe youtube

Dă-mi link la video, te rog.

vladutcornel commented on index.html:
```diff
@@ -0,0 +1,96 @@
+            <path d="M 23.857422 8.5 C 17.504717 8.5 11.602344 8.9526234 8.234375 9.65625 A 1.50015 1.50015 0 0 0 8.2128906 9.6621094 C 5.6754768 10.230693 3.2861897 12.048234 2.7832031 14.894531 A 1.50015 1.50015 0 0 0 2.78125 14.90625 C 2.394836 17.200265 2 20.190694 2 24.5 C 2 28.801151 2.3961903 31.712324 2.8847656 34.126953 C 3.4000756 36.889296 5.7342165 38.761817 8.3105469 39.337891 A 1.50 ...
+            </svg></a>
+    </div>
+    <h4>&#169 2024 Mondelez</h4>
```
Caracterele speciale trebuie să se termine cu `;`.

```suggestion
    <h4>&#169; 2024 Mondelez</h4>
```

Dar există caracter special mai sugestiv.
```suggestion
    <h4>&copy; 2024 Mondelez</h4>
```

vladutcornel commented on style.css:
```diff
@@ -0,0 +1,344 @@
+@media screen and (min-width: 600px) {
+    .hero-promo {
+        display: grid;
+        grid-template-columns: repeat (3, 1fr);
```
În CSS nu se pune spațiu înainte de `(`
```suggestion
        grid-template-columns: repeat(3, 1fr);
```

k4pn0 commented:
> > am urmat un video pe youtube
> 
> Dă-mi link la video, te rog.

https://youtu.be/dAIVbLrAb_U?si=3i26Wv0zQPPHVf55

vladutcornel commented on style.css:
```diff
@@ -0,0 +1,344 @@
+    justify-content: center;
+    align-items: center;
+}
+.hero-promo img {
```
Ai grijă că acest selector se potrivește și imaginilor pe care le ai în `.container-img`

vladutcornel commented on style.css:
```diff
@@ -0,0 +1,344 @@
+    fill: #f4f4f7;
+}
+
+@media screen and (min-width: 600px) {
```
Pe ecrane mai mari, ai conținutul mai lat decât ecranul. Pare să fie de la imaginile cu premii.

vladutcornel commented on style.css:
```diff
@@ -0,0 +1,344 @@
+    display: flex;
+    align-self: center;
+    justify-self: left;
+    max-width: 95vw;
```
Vezi dacă te ajută să pui și width
```suggestion
    max-width: 95vw;
    width: 100%;
```

vladutcornel commented on index.html:
```diff
@@ -0,0 +1,96 @@
+    </div>
+<!--            (=🝦 ༝ 🝦=)           -->
+<footer>
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="f4f4f7" fill-opacity="1" d="M0,256L60,218.7C120,181,240,107,360,74.7C480,43,600,53,720,85.3C840,117,960,171,1080,176C1200,181,1320,139,1380,117.3L1440,96L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z"></path></svg>
```
Cred că ai văzut că SVG-ul tău este invers. Poți să îi faci flip ori dintr-un editor, ori din CSS.

Oriunde i-ai face flip, eu aș zice că te-ar ajuta să îl pui altfel. Poate ca background la un element.

vladutcornel reviewed (changes requested):
Eu îți recomand să te concentrezi la chestiile pe care știi cum să le faci. Și ne uităm Marți la chestiile care îți dau bătăi de cap.

Mai bine faci mai puțin și bine.

k4pn0 commented:
Mulțumesc mult pentru feedback și răbdare. O sa le fac și trimit din nou în seara asta. Cred că o să mă concentrez pe homepage sa fie totul bine și responsive și să nu mai fiu haotică. Posibil sa te asaltez cu alte întrebări și mâine.

k4pn0 committed:
aproape final

Cum ziceam am facut ce am putut dar nu prea stiu cum il vei nota sau cum il prezint.

vladutcornel commented:
Să nu uiți să îmi dai URL-ul de deploy.
